src/r/app.R

```diff
@@ -1,12 +1,11 @@
 #install.packages("maptools")
 #install.packages("data.table")
-#install.packages("testthat")
```

Comment:
@seb231 Is there a reason you've removed the installation of the testthat package but not the others?